corehq/ex-submodules/phonelog/reports.py

```diff
@@ -73,7 +73,10 @@ class BaseDeviceLogReport(GetParamsMixin, DatespanMixin, PaginatedReportMixin):
     @property
     def headers(self):
         return DataTablesHeader(
-            DataTablesColumn("Date", span=1, sort_type=DATE, prop_name='date',
+            DataTablesColumn("Log Date", span=1, sort_type=DATE, prop_name='date',
+                             sort_direction=[DTSortDirection.DSC,
+                                             DTSortDirection.ASC]),
+            DataTablesColumn("Server Date", span=1, sort_type=DATE, prop_name='server_date',
```

Comment:
I could see this terminology being a little confusing. Could maybe change to Log Date/Log Submission Date to match some of our other reports?